### PR TITLE
use template literal on some few places

### PR DESCRIPTION
**Pull request by jimmywarting**
**Marked version:**

**Markdown flavor:** Markdown.pl|CommonMark|GitHub Flavored Markdown|n/a

## Description

- Fixes #### (if fixing a known issue; otherwise, describe issue using the following format)

## Contributor

- [ ] Test(s) exist to ensure functionality and minimize regression (if no tests added, list tests covering this PR); or,
- [ ] no tests required for this PR.
- [ ] If submitting new feature, it has been documented in the appropriate places.

## Committer

In most cases, this should be a different person than the contributor.

- [ ] CI is green (no forced merge required).
- [ ] Squash and Merge PR following [conventional commit guidelines](https://www.conventionalcommits.org/).

### CODE REVIEW

File: src/Renderer.js [hunk 1]
```diff
@@ -101,36 +94,36 @@ export class Renderer {
...
+    return `<tr>\n${content}</tr>\n`;
   }
 
   tablecell(content, flags) {
     const type = flags.header ? 'th' : 'td';
     const tag = flags.align
-      ? '<' + type + ' align="' + flags.align + '">'
-      : '<' + type + '>';
-    return tag + content + '</' + type + '>\n';
+      ? `<${type} align="${flags.align}">'`
```
Comment (UziTech) [inline src/Renderer.js:103]:
```suggestion
      ? `<${type} align="${flags.align}">`
```

**Review by UziTech — CHANGES_REQUESTED** [review]

**Comment by calculuschild** [comment]
Do template literals cause any noticeable slowdown?

**Comment by UziTech** [comment]
I don't see any difference when running `npm run bench` on master or this PR.

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**